docs/pages/blog/aggregation-functions.md

```diff
@@ -0,0 +1,111 @@
+      const sortedValue = params.values.sort((a = '', b = '') => a.localeCompare(b));
+      return sortedValue[0];
+    },
+    // The `label` defines what's displayed in the column header when this aggregation is active.
+    label: 'First Alphabetical',
+    // The `types` property defines which type of columns can use this aggregation function.
+    // Here, we only want to propose this aggregation function for `string` columns.
+    // If not defined, aggregation will be available for all column types.
+    columnTypes: ['string'],
+  };
+```
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/aggregation-functions/with-custom-functions.mp4" type="video/mp4" />
+</video>
+
+## Okay, I’m in! How can I get started?
+
+:::info
+First make sure that you're using `@mui/x-data-grid-premium` and version `5.15.0` or later.
+
+:::
+
+During the initial version of Aggregation functions, we’re publishing the feature as experimental, so we have some room to validate its API and the overall direction we’re taking with it.
+But the feature is stable, and the API will likely not change significantly.
+
+To activate the feature, please enabled it with the experimentalFeatures prop.
+
+```tsx
+<DataGridPremium experimentalFeatures={{ aggregation: true }} {...otherProps} />
+```
+
+And that's it! That enables the aggregation option on the column menu (available by default on columns of type number and string).
+
+## Use with tree data and more
+
+We already mentioned that you could customize functions and use aggregation with row grouping. But there are a lot of other possibilities to explore!
+
+Please check out the feature’s [full documentation](/x/react-data-grid/aggregation/) to get a better overview of everything it has to offer.
+
+## Row pinning
+
+Aggregation footer row is empowered by another new feature called [Row pinning](/x/react-data-grid/rows/#row-pinning).
+It enables you to pin rows at the bottom or at the top of your data grid.
+Pinned rows are visible at all times while user scrolls the grid vertically.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/aggregation-functions/row-pinning.mp4" type="video/mp4" />
+</video>
+
+Row pinning is available in Data grid Pro and Premium.
+
+Check out more details on its [documentation page](/x/react-data-grid/rows/#row-pinning).
+
+## Share your feedback 🗣
+
+**And if you’re feeling generous, we’re recruiting users to interview!**
+
+We hope you’re excited about the new features as we are!
+
+As usual, we’re glad to listen to any feedback. Please feel free to open new issues to report bugs or suggest improvements.
+
+Moreover, we’re planning the MUI X v6 release, and we’d love to hear more about your pain points and use cases.
+If you want to help us steer our product, please consider [contacting us](https://forms.gle/vsBv6CLPz9h57xg8A) to schedule a 30 minutes user interview session.
```

Comment:
The header looks strange, I thought it was an overlay bug at first

<img width="661" alt="Screenshot 2022-07-25 at 18 26 02" src="https://user-images.githubusercontent.com/3165635/180827817-7126b1b0-ca3f-45f6-9064-1e6d542a5859.png">

Reply:
Could you share the Google form with all@mui.com? It doesn't seem that I have access

Reply:
@oliviertassinari, you should have access now.
I've reverted the changes to the header image. I did a couple of experiments, but I wasn't sure about the result either.